docs/markdown/index.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: default
+title: Index
+nav_order: 1
+permalink: /
+---
+
+# Snowbridge Documentation
+{: .fs-9 }
+
+Welcome to the documentation site for Snowbridge, a trustless bridge system for bridging arbitrary data between Polkadot, Parachains and Ethereum.
+
+[Snowbridge Website](https://snowbridge.snowfork.com/){: target="_blank" .btn-purple .btn .fs-5 .mb-4 .mb-md-0 }
```

Comment:
These buttons should all be on 1-line for better visual appeal. Maybe remove the newlines between them to achieve that.